pkgs/applications/virtualization/singularity/generic.nix

```diff
@@ -281,6 +282,14 @@ in
         singularity = finalAttrs.finalPackage;
       };
     };
+    # Run the packages to test. E.g.
+    # nix run .#apptainer.manualTests.run-image-hello-cowsay
+    manualTests = {
```

Comment:
Oh! I too was wondering about how to better expose "nix run"-able tests: https://github.com/NixOS/nixpkgs/pull/295653#discussion_r1523986413

Wdyt?

Reply:
@SomeoneSerge It might require some consensus to define a place for these manual tests.

At least we expect that those test script packages be successfully built. Maybe we could put them inside `tests.manualTests` with `recurseIntoAttrs` to make the CI build them automatically.